cmd/loadtest/loadtest.go

```diff
@@ -1477,6 +1487,66 @@ func loadTestInscription(ctx context.Context, c *ethclient.Client, nonce uint64)
 	return
 }
 
+func loadTestBlob(ctx context.Context, c *ethclient.Client, nonce uint64) (t1 time.Time, t2 time.Time, err error) {
+	ltp := inputLoadTestParams
+
+	to := ltp.ToETHAddress
+	if *ltp.ToRandom {
+		to = getRandomAddress()
+	}
+
+	amount := ltp.SendAmount
+	chainID := new(big.Int).SetUint64(*ltp.ChainID)
+	privateKey := ltp.ECDSAPrivateKey
+
+	gasLimit := uint64(21000)
+	gasPrice, gasTipCap := getSuggestedGasPrices(ctx, c)
+	blobFeeCap := uint64(1000000000) // 1eth
+
+	// Initialize blobTx with blob transaction type
+	blobTx := ethtypes.BlobTx{
+		ChainID:    uint256.NewInt(chainID.Uint64()),
+		Nonce:      nonce,
+		GasTipCap:  uint256.NewInt(gasTipCap.Uint64()),
+		GasFeeCap:  uint256.NewInt(gasPrice.Uint64()),
+		BlobFeeCap: uint256.NewInt(blobFeeCap),
+		Gas:        gasLimit,
+		To:         *to,
+		Value:      uint256.NewInt(amount.Uint64()),
+		Data:       nil,
+		AccessList: nil,
+		BlobHashes: make([]common.Hash, 0),
+		Sidecar: &types.BlobTxSidecar{
+			Blobs:       make([]kzg4844.Blob, 0),
+			Commitments: make([]kzg4844.Commitment, 0),
+			Proofs:      make([]kzg4844.Proof, 0),
+		},
+	}
+	// paseBlobVariables will take in the blobTx struct and append values to blob transaction specific keys
+	err = parseBlobCommitment(&blobTx)
```

Comment:
Maybe here you could update the logic to do:

- Generate random blob data
- Generate blob commitment for this piece of data
- Append the blob commitment to the blob tx

I think it would be clearer for the reader

Reply:
Those steps are being done in the `blob.go` file, and I thought it would be cleaner to abstract it away. But I agree the function names and logic may not be very clear, so I'll change the function names and try to add detailed comments for the logic to account for this.. hopefully it'll be clearer. 
Thank you for the feedback!